builtin/providers/cloudstack/resource_cloudstack_instance.go

```diff
@@ -82,6 +82,11 @@ func resourceCloudStackInstance() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"project_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  nil,
```

Comment:
This should be removed as it servers no purpose here. `Default` is only useful when you want to assign an actual default value if the user didn't specify anything. If the user doesn't specify a value the types zero value will be used. And in Go for strings that is and empty string so `""` and not `nil` (in Go a string cannot be `nil`).